### PR TITLE
Add absent members to the meeting protocolData

### LINKED ISSUE #4875: Entschuldigte Teilnehmer in procol-JSON einfügen
Im Moment werden entschuldigte Teilnehmer nicht im JSON der Sitzung eingefügt, damit kann man sie in Protokollen nicht separat auflisten. 

Die abwesenden/entschuldigten Mitglieder sollen separat aufgelistet werden. In https://github.com/4teamwork/opengever.core/blob/master/opengever/meeting/protocol.py#L78 werden abwesende in die `JSON`-Daten eingefügt. Die abwesenden sollen z.B. unter dem key `absent` aufgelistet werden, Beispiels-`JSON`-Snippet der Teilnehmer:

```
    "participants": {
        "other": [
            "Heidi"
        ], 
        "members": [
            {
                "lastname": "Meier", 
                "fullname": "Meier Hans", 
                "role": null, 
                "email": null, 
                "firstname": "Hans"
            }
        ],
        "absent:" [
            {
                "lastname": "M\u00fcller", 
                "fullname": "M\u00fcller Peter", 
                "role": "Scheff", 
                "email": null, 
                "firstname": "Peter"
            }
        ]
    }, 

```

Tipp: Um die Abwesenden herauszufinden muss man vermutlich zuerst mittels `Membership.query.for_meeting` alle möglichen Teilnehmer herausfinden, die Abwesenden sind dann das Delta zu `Meeting.participants`.

Bitte auch Docs aktualisieren: https://docs.onegovgever.ch/admin-manual/meeting/mergefields/#metadaten-zur-sitzung.

### PR DESCRIPTION
**Pull request by njohner**
* We add the absent members to the meeting `protocolData`.
* We also add the absents to the sablon templates in the example content.
* Finally we corrected a few broken fields in the sablon templates in the example content. There was notably a field `{ SUBJECT  \* MERGEFORMAT }` in several templates, which is missing the `MERGEFIELD =` and was therefore not working. Moreover there is no `SUBJECT` in the `protocolData`, so I simply removed this field from the templates.

Here snapshots from the output docs files when filling the templates with the `SAMPLE_MEETING_DATA`:
**Top part from protocol_kopf.docx**
protokollkopf

**Top part from protocol.docx**
protokoll

Before we had:

**Top part from protocol_kopf.docx**
protokoll_kopf

**Top part from protocol.docx**
protokoll

resolves #4875

### CODE REVIEW

File: docs/public/admin-manual/meeting/mergefields.rst [hunk 1]
```diff
@@ -83,6 +79,12 @@ Metadaten zur Sitzung:
   Liste wird typischer-weise iteriert, um die Namen darzustellen (siehe weiter
   unten).
 
+- ``participants.absents``
```
Comment (deiferni) [inline docs/public/admin-manual/meeting/mergefields.rst]: a detail, but wouldn't `participants.absent` be better 🤔?

File: opengever/meeting/protocol.py [hunk 2]
```diff
@@ -48,26 +48,33 @@ def add_protocol_type(self):
...
-
-        return participants
+            members.append(self.get_member_data(participant, membership))
+        return members
+
+    def get_absent_members(self):
+        # Now we add the members that are not attending the meeting
+        absents = []
+        participants = set(self.meeting.participants)
+        for membership in Membership.query.for_meeting(self.meeting):
```
Comment (deiferni) [inline opengever/meeting/protocol.py]: could we move building the list of absent participants to the meeting, so we can ask the meeting for its absent participants? could also be relevant for https://github.com/4teamwork/opengever.core/pull/5101.

File: docs/public/admin-manual/meeting/mergefields.rst [hunk 3]
```diff
@@ -83,6 +79,12 @@ Metadaten zur Sitzung:
...
+
+  Liste aller Mitglieder die zur Sitzung abwesend sind (Liste von Participant).
+  Über diese Liste wird typischer-weise iteriert, um die Namen darzustellen
+  (siehe weiter unten).
```
Comment (deiferni) [inline docs/public/admin-manual/meeting/mergefields.rst]: (siehe weiter unten) -> link 😄?

File: docs/public/admin-manual/meeting/mergefields.rst (hunk 1 above)
Comment (njohner) [reply docs/public/admin-manual/meeting/mergefields.rst]: well we use the plural for `participants.members`, why not for `absents`. A bit surprising, but it's the correct plural.

Comment (njohner) [reply docs/public/admin-manual/meeting/mergefields.rst]: I could also propose `absentees`. I think the word is outdated, but maybe conveys the idea better in this case?

**Comment by njohner** [comment]
Ok ready for rereview. I have kept a separate commit for the introduced changes in the code, moving the creation of the `absents` list to the meeting.

File: docs/public/admin-manual/meeting/mergefields.rst (hunk 1 above)
Comment (deiferni) [reply docs/public/admin-manual/meeting/mergefields.rst]: oh, yes please use `absentees`, that sounds like the correct word 👍.

**Comment by njohner** [comment]
Ok I renamed `absents` to `absentees` and readapted the example templates accordingly.
This should be ready now.

**Pull request merged**